private/buf/cmd/buf/command/alpha/repo/reposync/reposync.go

```diff
@@ -61,11 +59,12 @@ func NewCommand(
 	return &appcmd.Command{
 		Use:   name,
 		Short: "Sync a Git repository to a registry",
-		Long: "Sync a Git repository's commits to a registry in topological order. " +
+		Long: "Sync commits in a Git repository to a registry in topological order. " +
 			"Only commits in the current branch that are pushed to the 'origin' remote are processed. " +
-			"Syncing all branches is possible using '--all-branches' flag." +
-			// TODO rephrase in favor of a default module behavior.
-			"Only modules specified via '--module' are synced.",
+			"Syncing all branches is possible using '--all-branches' flag. " +
+			"By default a single module at the root of the repository is assumed, " +
+			"for specific module paths use the '--module' flag. " +
```

Comment:
nit: we can do this in a follow-up, but we should provide a more detailed example of how the `--module` flag is used.

Reply:
Sure, although that I think it's already addressed in the `--module` flag docs. I'll check and add it in upcoming PRs if it's not there.